documentation/source/development/reference/gpu-selection.rst

```diff
@@ -1,25 +1,10 @@
 GPU selection mechanism
 =======================
 
-- Every graphics card is a gpu, but not every GPU is a (discrete) graphics card.
-- For example, some GPUs are integrated into the main processor.
-- If a certain GPU is not visible for Vulkan (vkEnumeratePhysicalDevices `<https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/vkEnumeratePhysicalDevices.html>`__), it does not support Vulkan.
-- If multiple GPUs are available on the system, the user can either specify which one should be used or let the engine select an appropriate GPU automatically.
-- The user can specify which GPU to prefer by using ``--gpu <index>`` as command line argument.
-- The ``<index>`` is the array index, **starting from 0**.
-- The engine will verify that ``<index>`` is a valid value.
-- The engine will check if the GPU is suitable for the engine's purposes. For example we might require GPUs to support geometry shaders in the future.
-- If no preferred gpu was specified by the user, the specified gpu index is invalid, or the gpu is unsuitable, **automatic gpu selection rules** apply.
-
-Automatic GPU selection rules
------------------------------
-
-- Automatic gpu selection is part of the so called **settings decision maker**.
-- The method which ultimately selects the gpu is ``decide_which_graphics_card_to_use`` in `settings_decision_maker.cpp <https://github.com/inexorgame/vulkan-renderer/blob/main/src/vulkan-renderer/settings_decision_maker.cpp>`__.
-- Note that we are using a C++17 feature which is called `std::optional <https://en.cppreference.com/w/cpp/utility/optional>`__.
-- This means if no gpu could be found after all, ``std::nullopt`` will be returned.
-- The automated GPU selection mechanism will prefer the discrete GPU over the integrated GPU when exactly 2 GPUs are available.
-
-.. image:: gpu_selection_mechanism.svg
-    :width: 1000
-    :alt: GPU selection mechanism
+- In Vulkan API, the word physical device is a more general word for all types of graphics cards, integrated gpus and more
+- If multiple physical devices are available on the system, Inexor engine is able to pick the most suitable one automatically
+- The user can specify a preferred graphics card index with the command line argument ``--gpu <index>`` (starting with index ``0``!)
+- If a preferred index is specified, the engine will verify if the index is valid and pick the physical device if it is suitable
+- If the physical device specified by the user is not suitable because of technical reasons, automatic selection rules apply
+- The engine calculates a score for every available physical device based on the device type and total video memory size
+- If no physical devices are available or no suitable physical device could be chosen, an exception is thrown
```

Comment:
Here you document the exception but not in source :D

Reply:
Done.